AirBench/Controllers/BenchController.cs

```diff
@@ -67,13 +67,23 @@ public ActionResult Index ()
         {
             BenchRepository repository = new BenchRepository(context);
             ReviewRepository repo = new ReviewRepository(context);
+            LoginRepository Repository = new LoginRepository(context);
             List<Bench> myBenches = repository.GetBenchList();
             List<BenchList> myBenchList = new List<BenchList>();
             foreach(var bench in myBenches)
             {
+                User user = Repository.GetById(bench.CreatorUserId);
+                var name = user.Name.Split(' ');
                 var reviews = repo.GetReviewList(bench.Id);
                 BenchList currentBench = new BenchList();
                 currentBench.CreatorUserId = bench.CreatorUserId;
+                if (!(name.Count() > 1)) {
+                    currentBench.Name = name[0];
+                }
+                else
+                {
+                    currentBench.Name = name[0] + ' ' + name[1].ToCharArray()[0] + '.';
```

Comment:
Alternatively, you could use `name[1].Substring(0, 1)` to get the first character of the last name.